tests/test-verify-pool-members.yml

```diff
@@ -70,6 +70,23 @@
     loop_var: pv
   when: storage_test_pool.type == 'lvm'
 
+- name: Check that blivet supports PV grow to fill
+  ansible.builtin.script: >-
+    scripts/does_library_support.py
+    blivet.formats.lvmpv.LVMPhysicalVolume.grow_to_fill
+  register: grow_supported
+  changed_when: false
+
+- name: Verify that PVs fill the whole devices when they should
+  include_tasks: verify-pool-member-pvsize.yml
+  loop: "{{ _storage_test_pool_pvs | default([], true) }}"
```

Comment:
You shouldn't need to use the form `default([], true)` here - just `default([])` should work - if `_storage_test_pool_pvs` is defined, it should be a `list` - if it is an empty list (which evaluates to `false` in a boolean context, which is what the `, true` is for in the `default` filter), then nothing will be done - the loop will not be executed.

Reply:
Fixed.
(Removed unnecessary `true`)